src/benchmark-utils.ts

```diff
@@ -9,7 +9,23 @@ import {TensorFlowBenchmark} from './tfjs-benchmark';
 export interface BenchmarkResult {
   framework: string;
   backend: string;
-  duration: number;
+  actualBackend :string;
+  avg: number; 
+  pt50: number; 
```

Comment:
I'm not sure whether 'p50' is common or 'pt50'. at least, onnxruntime_perf_test tool is using 'p50'.